js/script.js

```diff
@@ -0,0 +1,199 @@
+let mainArray = [{
```

Comment:
const movies

let only for variables which we will have changeable

and we have array with specific data. our objects are information about films. i recommend use meaningful names